pkgs/applications/networking/mailreaders/dodo/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, lib, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dodo";
+  version = "2022-08-13";
```

Comment:
This should be `unstable-YYYY-MM-DD` where YYYY-MM-DD is the date of the commit.